### ISSUE TITLE
508-defect-3: Edit links should be removed from the issue summary page

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#53975]
## Point of contact

joshkimux

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

## Details

This _may_ be a defect-3 depending on how its interpreted and the design intent ( @eileen-coforma for awareness ✨ )

At first glance, these edit links (see screenshot) appear to allow a user to edit their issues.

<img width="1490" alt="Screenshot 2023-02-21 at 4 09 05 PM" src="https://user-images.githubusercontent.com/14154792/220459388-96de4ce2-bca2-49dc-a28d-7a48022d524f.png">

However, both of these links (and the back button) simply return the user back to the select an issue page to edit their _selection_ (as opposed to the specific issue).

<del>

If the intent is to allow the user to go directly to the edit issue page, (which aligns with the language "edit"), the links should be unique e.g. "Edit _PTSD_"

If the intent is to simply let the user change their selection, we should
- remove the links and let the user rely on the back button OR
- provide _one_ link that is clearly labelled as "Edit selection"

Another issue related to this (which I haven't drafted a ticket on as it's a `defect-3`) is the language of change vs. edit. On the issue selection page, a user can go the _edit_ issue page by clicking on "change" (which is a mismatch of the semantics).

</del>

 

## Reproduction steps

1. [Follow the login instructions here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/52046)
2. Add multiple issues on the [issue summary page](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/issue-summary)
3. On the issue summary page, inspect each of the edits and confirm none of them are uniquely labelled.
4. Click on either of the edit options, confirm both take the user back to the issue selection page.

## Success criteria

- [x] Confirm with @eileen-coforma the design intent of the edit links. Specifically, are they meant to let the user edit their selection, or edit the issue itself?
- [x] Reduce this ticket to a `defect-3`
- [x] Remove both edit links (guidance by @eileen-coforma )

## References, articles, or WCAG support

1.
2.
3.
...

## Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
There is a bit of complexity to this... if the Ankylosis of the knee issue was a loaded in contestable issue, then they can only change the checkbox state on the contestable issue page. Whereas if the issue was manually added, we could go back to the "Add issue" page and allow them to edit the values within the issue, but they can only change the checkbox state and remove the issue on the contestable issue page.

![contestable issue page showing ankylosis of knee as an API loaded issue and test as a second manually added issue. A change link and remove button are only shown for the second issue](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Supplemental-Claims/engineering/new-list-loop.png?raw=true)

**Comment by joshkimux** [comment]
@Mottie , this is _amazing context. This feels like we should either
- pivot to "change selection" language OR
- simply remove "edit" on the issue summary page OR
- provide "edit" _only_ for uer entered (and not API entered) issues
  - on the caveat-- do we have any anticipation that users might be confused if they can edit one thing but not the other?

**Comment by eileen-coforma** [comment]
We are actually running into this issue in our studies. A few veterans thought it would edit the decision date (proximity relation), where as the intent is that they can edit which issues they want to add on the summary page.

I am fine with removing the edit link altogether. 
1. It's a duplicate function to what the back button can do.
2. For custom entries, I don't think the extra click to go through in the flow would cause much friction.

Unless there are any other reasons I should take into consideration @Mottie @joshkimux

**Comment by joshkimux** [comment]
Love a good usability-testing informed design decision ✨

**Comment by joshkimux** [comment]
Deal! I'll update the SC here!

**Comment by joshkimux** [comment]
Updated @Mottie , this is now a `defect-level-3` with updated success criteria

**Comment by saderagsdale** [comment]
@rocio-desantiago We're removing the edit links from the design.

**Comment by joshkimux** [comment]
Confirming this is complete! Way to go @rocio-desantiago 💯
